platform/comms/housekeeping.c

```diff
@@ -45,6 +45,16 @@ SAT_returnState hk_report_parameters(HK_struct_id sid, tc_tm_pkt *pkt) {
 	i = 1;
         cnv32_8(HAL_sys_GetTick(), &pkt->data[i]);
         i += sizeof(uint32_t);
+        /* FIXME: Get the actual reboot code */
+        b = comms_stats.rst_src;
```

Comment:
Fixed the boot event byte at 43c38f72a4eeaa6d8442a0ded86afcda6b5cff9a. Now we miss only the last assertion code